.env

```diff
@@ -10,10 +10,10 @@ TIMEZONE="Melbourne"
 DEFAULT_COUNTRY_CODE="AU"
 
 # Locale for translation.
-LOCALE="en"
+LOCALE="en_AU"
```

Comment:
As this affects all environments, I had a think:

- It shouldn't matter for servers, they would all have config to override this. 
- It shouldn't affect devs much, because any new translation keys can still be added in en.yml, and the fallback will work. But what if they're confused and add new translations to en_AU?
- Apparently it didn't affect specs. I would have thought some differences would pop up. Maybe they would in the future.

It seems strange that en_AU is the default instead of en. I can imagine someone changing it back again just because it looks wrong.

🤔 If en_AU is the default for dev/test, then en.yml is sort of redundant. Of course it still functions as the fallback, and is the source of truth for Transifex.

Reply:
> If en_AU is the default for dev/test, then en.yml is sort of redundant.

en_AU is still a production locale which is edited in Transifex while en.yml is the one locale changed by devs.

We could introduce a test locale which would be a bit more robust. At the moment, the Australian team could change something that breaks specs. It would be less realistic though. :thinking: 

Okay, what do you think about this proposal:

* en_AU in .env.development, it's more realistic and we got other Australian data like default country and currency already.
* en_TEST in .env.test.

Reply:
That sounds good 👍 

I guess en_TEST may be empty to begin with, until we add a specific test for it.

Reply:
So, just to make sure I understood this correctly: the use-case here for specs is to have the empty file, so we can select it for a give system spec, and assert that no error is there, which demonstrates the fall-back to en.yml works correctly. This sounds good.

> I guess en_TEST may be empty to begin with, until we add a specific test for it.

As for this, having valid keys and translations would be useful to test that the default selection works correctly in the test environment, if I get this right.

Reply:
Yes, you got that right. It would be good to also test that the current locale is actually used and not just the fallback. :wink: